--- evdev       2018-11-30 13:53:13.230688508 +0100
+++ evdev.mod   2018-11-30 13:52:58.274774722 +0100
@@ -1136,7 +1136,8 @@
   lv5:lwin_switch_lock_cancel     =       +level5(lwin_switch_lock_cancel)
   lv5:rwin_switch_lock_cancel     =       +level5(rwin_switch_lock_cancel)
   parens:swap_brackets   =   +parens(swap_brackets)
-  
+  mapple:super = +mapple(super)
+
 
 ! option       =       compat
   grp_led:num          =       +lednum(group_lock)
